tests_e2e/test_suites/images.yml

```diff
@@ -81,7 +81,10 @@ images:
          AzureChinaCloud: []
    centos_610: "OpenLogic CentOS 6.10 latest"
    centos_79: "OpenLogic CentOS 7_9 latest"
-   centos_82: "OpenLogic CentOS 8_2 latest"
+   centos_82:
+      urn: "OpenLogic CentOS 8_2 latest"
+      vm_sizes:
+         - "Standard_B2s"
```

Comment:
agent hang was reported for redhat+ in this vm size

Reply:
could you add this as a comment in the file itself? thanks